debian/tests/integrationtests

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-if [ "$(uname -i)" = 'armv7l' ]; then
-    echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
-fi
+echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
That sounds like a good idea. I'm still not sure if this will fix it though. The only thing that makes sense with this is error is that something changed in zesty.